sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -931,7 +931,16 @@ def test_pridect_signature(self, estimator_class):
         assert "X" in sig.parameters.keys()
         assert "y" in sig.parameters.keys()
 
+    def _check_global_tag(self, estimator_instance):
+        global_forecast = estimator_instance.get_tag(
+            "capability:global_forecasting", tag_value_default=False, raise_error=False
+        )
+        return global_forecast
+
     def test_global_forecasting_multiindex_hier(self, estimator_instance):
+        if not self._check_global_tag(estimator_instance):
```

Comment:
why are we adding these, could you explain?

Reply:
If `broadcasting==True`, the global forecasting ability will be turned off. It has been included in [ENH] Improve TestAllGlobalForecasters #6845. As #6845 is merged now, the changes should disappear after merging main.